plugin/Gui/AdminPageFormSettingsContact.php

```diff
@@ -114,6 +114,9 @@ protected function buildForms()
 
 		if ($this->_showCreateAddress) {
 			$pInputModel = $pInputModelBuilder->build(InputModelDBFactoryConfigForm::INPUT_FORM_CREATEADDRESS);
+			$txtHint = __( 'If the contact form is on an estate page, you can link the created address to that estate by opening the email you receive in onOffice enterprise or using the', 'onoffice-for-wp-websites') 
```

Comment:
Please don't split the text into fractions, translate the whole sentence instead.

```php
$link = sprintf('<a href="%s">%s</a>', esc_attr(__("https://de.enterprisehilfe.onoffice.com/category/additional-modules/request-manager/?lang=en", 'onoffice-for-wp-websites')), esc_html(__("request manager", 'onoffice-for-wp-websites')));
$txtHint = sprintf(__( 'If the contact form is on an estate page, you can link the created address to that estate by opening the email you receive in onOffice enterprise or using the %s.', 'onoffice-for-wp-websites'), $link);
```
Background: other languages may require the link to be placed in a different position than the end of the sentence.